python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -383,6 +386,14 @@ def run(self, dataRef, selectDataList=[]):
 
         return pipeBase.Struct(coaddExposure=coaddExp)
 
+    def makeSupplementaryData(self, dataRef, selectDataList):
+        """!
+        \brief Make additional inputs to assemble() specific to subclasses. 
+
+        To be implemented by subclasses if needed.
```

Comment:
Could you give an example of the sort of supplementary data that could be provided, or that might be needed?

Reply:
(Picky) Watch for trailing whitespace (line 391)